test/src/test/java/org/corfudb/integration/CorfuReplicationClusterConfigIT.java

```diff
@@ -1630,9 +1630,6 @@ public void testBackupRestoreWorkflow() throws Exception {
         updateTopology(activeCorfuStore, DefaultClusterManager.OP_BACKUP);
         log.info("Change the topology!!!");
 
-        TimeUnit.SECONDS.sleep(shortInterval);
```

Comment:
Yeah depending on sleep would make the code race prone. I'm wondering about the motivation behind this sleep.

Reply:
I thought when this sleep was introduced, the purpose was to give the new topology time to take effect (writing to test purpose config table -> DefaultClusterManager update topology -> topology change event discovered by CorfuReplicationDiscoveryService).

From local running I never hit the flakiness of this test, but we did observe several failures in GitHub Action. I guess this could be a possible reason as if DefaultClusterManager hasn't updated topology, and the new forced snapshot sync has been requested, LR will be performed by the original source cluster instead of the backup cluster